samples/AdministrationApi/dockerfile

```diff
@@ -1,6 +1,6 @@
 # escape=`
 
-FROM microsoft/iis
+FROM mcr.microsoft.com/windows/servercore/iis:windowsservercore-ltsc2019

```

Comment:
This is using a specific tag and not the latest as the original intended.

Reply:
Ok, I'm going to update to use `latest` tag now.

Do you want to also update other dockerfile to use `latest` tag?

Reply:
I have updated to add a commit to use default (latest) tag for all 3 dockerfile samples.